### ISSUE TITLE
Show point for current warning

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#7701]
It would be best to put a red flashing marker at the point of conflict,
![20003-11](https://user-images.githubusercontent.com/1230959/84553869-fac94880-ad47-11ea-903d-dcd0674af74f.jpg)
as it is not always clear from simply giving road name (if any!) exactly what intersection is being discussed.

**Comment by jidanni** [comment]
And for identically named features,
![6317-13](https://user-images.githubusercontent.com/1230959/84554034-a83c5c00-ad48-11ea-9a90-05bed22ec2d2.jpg)
it is a gamble to guess which one is being discussed.

**Comment by jidanni** [comment]
Yes I am only talking about warnings that refer to points (intersections of roads A and B).

**Comment by jidanni** [comment]
Yes road A is in red, but road B is only in purple if our mouse is hovering over the item in the yellow list. Hovering anywhere else removes the purple.

**Comment by quincylvania** [comment]
Sounds reasonable. I often run into similar situations where two features cross multiple times and it's unclear which crossing the issue corresponds to.
